tensorcircuit/circuit.py

```diff
@@ -916,12 +922,14 @@ def expectation(
 def expectation(
     *ops: Tuple[tn.Node, List[int]],
     ket: Tensor,
+    d: Optional[int] = None,
```

Comment:
better d after bra

Reply:
also be consistent with the var name for different function across tc, d or dim?